exonum-light-client/src/main/java/com/exonum/client/ExplorerApiHelper.java

```diff
@@ -20,15 +20,16 @@
 import static com.exonum.client.ExonumApi.JSON;
 import static java.util.stream.Collectors.toList;
 
+import com.exonum.binding.common.blockchain.ExecutionStatuses;
 import com.exonum.binding.common.blockchain.TransactionLocation;
-import com.exonum.binding.common.blockchain.TransactionResult;
 import com.exonum.binding.common.hash.HashCode;
 import com.exonum.binding.common.message.TransactionMessage;
 import com.exonum.client.response.Block;
 import com.exonum.client.response.BlockResponse;
 import com.exonum.client.response.BlocksResponse;
 import com.exonum.client.response.TransactionResponse;
 import com.exonum.client.response.TransactionStatus;
+import com.exonum.core.messages.Runtime.ExecutionStatus;
```

Comment:
how does it come here? Is it a part of `exonum-java-binding-common`? I mean `exonum.core.` package

Reply:
>  Is it a part of exonum-java-binding-common?

Yes, it is, but, ideally, it mustn't be a part of it, but a dependency of it. I think we shall extract the messages into a separate module (or modules, if we believe that some messages are not needed everywhere, e.g., supervisor messages, or merkledb messages) that will have not just have its own package, but its own release cadence (since the main reason for their release is a new Exonum release).